tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "@salesforce/dev-config/tsconfig",
   "compilerOptions": {
-    "outDir": "lib"
+    "outDir": "lib",
+    "rootDir": "src"
```

Comment:
This was preventing commands from being found during `snapshot:generate` unless the project was built.

Example from `plugin-schema`:

BEFORE
```
config:@salesforce/plugin-schema loading IDs from /Users/ewillhoit/dev/sandbox/po/plugin-schema/lib/commands +0ms
  config:@salesforce/plugin-schema found commands [] +3ms
```

AFTER
```
config:@salesforce/plugin-schema loading IDs from /Users/ewillhoit/dev/sandbox/po/plugin-schema/src/commands +0ms
  config:@salesforce/plugin-schema found commands [ 'force:schema:sobject:describe', 'force:schema:sobject:list' ] +5ms
```

Reply:
if it needs to be *everywhere* then it should be added to `dev-config`

Reply:
It doesn't look like it is needed _everywhere_. It was already set on all but 2 plugins but it is only set on a handful of libraries and utilities.